scripts/ipxe-dts.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+SCRIPTDIR=$(readlink -f "$(dirname "$0")")
+
+error_exit() {
+    _error_msg="$1"
+    echo "$_error_msg"
+    exit 1
+}
+
+error_check() {
+    _error_code=$?
+    _error_msg="$1"
+    [ "$_error_code" -ne 0 ] && error_exit "$_error_msg : ($_error_code)"
+}
+
+userIP=$(ip route get 1 | sed 's/^.*src \([^ ]*\).*$/\1/;q')
+error_check "cannot get user IP"
+
+distroVersion=$(cat "$SCRIPTDIR"/../meta-dts-distro/conf/distro/dts-distro.conf | grep -F "DISTRO_VERSION" | awk -F'"' '{print $2}')
+error_check "cannot get distro version from 'meta-dts/meta-dts-distro/conf/distro/dts-distro.conf'"
+
+cd $SCRIPTDIR
+error_check "cannot cd to 'meta-dts/scripts'"
+
+mkdir -p ipxe-files
+error_check "cannot make directory 'ipxe-files'"
+
+cd ipxe-files
+error_check "cannot cd to 'ipxe-files'"
+
+wget --output-document="bzImage-v$distroVersion" "https://boot.dasharo.com/dts/v$distroVersion/bzImage-v$distroVersion"
+error_check "cannot wget the bzImage v$distroVersion from https://boot.dasharo.com/dts/v$distroVersion/bzImage-v$distroVersion"
+
+wget --output-document="dts-base-image-v$distroVersion.cpio.gz" "https://boot.dasharo.com/dts/v$distroVersion/dts-base-image-v$distroVersion.cpio.gz"
+error_check "cannot wget the cpio.gz v$distroVersion from https://boot.dasharo.com/dts/v$distroVersion/dts-base-image-v$distroVersion.cpio.gz"
+
+port=${1:-9000}
+
+cat <<EOF >dts.ipxe
+#!ipxe
+#
+kernel http://$userIP:$port/bzImage-v$distroVersion root=/dev/nfs initrd=http://$userIP/dts-base-image-v$distroVersion.cpio.gz
+initrd http://$userIP:$port/dts-base-image-v$distroVersion.cpio.gz
+module http://$userIP:$port/test.sh /sbin/ipxe-commands mode=755
```

Comment:
let's call it `commands.sh` then in the documentation we could point that this file could be modified to run custom commands; we could then push that `commands.sh` script to meta-dts instead of creating test script while running this one

Reply:
@TomaszAIR done [here](https://github.com/Dasharo/meta-dts/pull/96/commits/2e3f91dbf58df3ce1f14e9c0ec2ca18098678120#diff-50ccba31fdf15d04ffc9cadbcbbfc4ceebd19e4076523602a1ee758e9a72cc68R47-R48), the `commands.sh` script gets copied everytime the `dts-ipxe.sh` script is ran.